code update for sellerConfig,[ATPRule +SafetyStock=inventoryAvailabil…

sheeladubey opened the pull request:
code update for sellerConfig,[ATPRule +SafetyStock=inventoryAvailability],inventoryOrganization,Returnitems

sheeladubey commented:
code update for sellerConfig,[ATPRule +SafetyStock=inventoryAvailability],inventoryOrganization,Returnitems

kothapallig commented on src/main/java/com/gsicommerce/romui/selenium/pages/ordermanagement/CommonElementsPage.java:
```diff
@@ -178,9 +174,9 @@ public static int getTotalRows() throws Exception {
 	}
 
-	public static void clickSelectLink(int row, int col, int divchild, int buttonchild) throws Exception {
+	public static void clickSelectLink(int row, int col) throws Exception {
```
Is it like you reverted the original code here???

sheeladubey replied:
no gopi.Created new branch with name -sellerConfig,inventoryAvailability,inventoryOrganization and pushed changes.I have made changes in CommonElementsPage.java file- method clickSelectLink .I have put else condition here to work for my code.Please review and let me know 

public void clickButton(int row, int col,int divchild,int buttonchild) {
		if(divchild==0 && buttonchild==0) {
		_xpath = String.format(".//tr[" + row + "]/td[" + col + "]/div/button", row, col);
		
		}else
		{
			_xpath = String.format(".//tr[" + row + "]/td[" + col + "]/div["+divchild+"]/button["+buttonchild+"]", row, col);
		}

kothapallig commented on src/main/java/com/gsicommerce/romui/selenium/utilities/Common.java:
```diff
@@ -586,22 +578,4 @@ public static void clickonCalendarDate(WebElement el, int days) {
 			el.sendKeys(selectDate(days));
```
from where el is read?

sheeladubey replied:
method clickonCalendarDate() already reviewed in last review  session.el is getting called in pipelinepage.java file.
New method need to review is waitForPageLoaded().

sheeladubey commented:
gopi added new methed waitForPageLoaded() in common.java.this is change in common code.Also added new methods like clickOnCancelBtn and clickDivChildSpanLink and modified clickDivSpanLink in CommonElementsPage.java

sheeladubey commented:
Gopi-Common.java file having -waitForPageLoaded() method  added newly.
CommonElementPage.java having clickOnCancelBtn and clickDivChildSpanLink  added newly and clickSelectLink() modified with else statement .
